specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-02-01-preview/elastic.json

```diff
@@ -2013,7 +2035,7 @@
           "description": "Results of a list operation.",
           "type": "array",
           "items": {
```

Comment:
is this going to be a list of versions? can this directly reference ElasticVersionListProperties?